tests/defaultbrowsercontext-1.spec.ts

```diff
@@ -140,6 +140,11 @@ it('should support userAgent option', async ({server, launchPersistent}) => {
   expect(request.headers['user-agent']).toBe('foobar');
 });
 
+it('should support platform option', async ({launchPersistent, browserName}) => {
```

Comment:
If we want to push cleaner tests
```suggestion
it('should support platform option', async ({launchPersistent}) => {
```